src/applications/gqrx/mainwindow.cpp

```diff
@@ -1654,6 +1655,8 @@ void MainWindow::forceRxReconf()
  */
 void MainWindow::on_actionDSP_triggered(bool checked)
 {
+    emit receiverStatusChanged(checked);
```

Comment:
We could simply call setReceiverStatus(checked) instead and save the declaration of the signal.